pyquil/noise_gates.py

```diff
@@ -0,0 +1,100 @@
+from pyquil.quilbase import Gate
```

Comment:
If everything in this module is private except for the imports and `THETA`, should it not be at the package top level?